rust/parquet/Cargo.toml

```diff
@@ -41,6 +41,7 @@ chrono = "0.4"
 num-bigint = "0.3"
 arrow = { path = "../arrow", version = "3.0.0-SNAPSHOT", optional = true }
 base64 = { version = "*", optional = true }
+serde_json = { version = "1.0", features = ["preserve_order"] }
```

Comment:
Better to make this one optional and gated.

Reply:
I agree